### PR TITLE
Chore/fix false positive test

### PR DESCRIPTION
**Pull request by anastasiyaig**
Fix test on checking member's role for channel

### CODE REVIEW

File: tests/communities/test_communities_channels.py [hunk 1]
```diff
@@ -90,19 +91,25 @@ def test_member_role_cannot_add_edit_and_delete_channels(main_screen: MainWindow
         with step('Right-click a channel on the left navigation bar'):
             community_screen.left_panel.right_click_on_panel()
         with step('Verify that context menu does not appear'):
-            assert not ContextMenu().is_visible
+            assert ContextMenu().is_visible is False, \
+                f"Context menu should not appear"
 
-    with step('Verify that member cannot edit and delete channel'):
+    with (step('Verify that member cannot edit and delete channel')):
```
Comment (Valentina1133) [inline tests/communities/test_communities_channels.py:97]: Not needed brackets before step

File: tests/communities/test_communities_channels.py [hunk 2]
```diff
@@ -90,19 +91,25 @@ def test_member_role_cannot_add_edit_and_delete_channels(main_screen: MainWindow
...
+            more_options = community_screen.tool_bar.open_more_options_dropdown()
         with step('Verify that edit item is not present in context menu'):
-            assert not more_options_dropdown.is_edit_item_visible()
+            assert more_options.is_edit_channel_option_present() is False, \
+                f'Edit channel option is present when it should not'
         with step('Verify that delete item is not present in context menu'):
-            assert not more_options_dropdown.is_delete_item_visible()
+            assert more_options.is_delete_channel_option_present() is False, \
+                f'Delete channel option is present when it should not'
+#
```
Comment (Valentina1133) [inline tests/communities/test_communities_channels.py:115]: No newline at the end of file and not needed #

**Review by Valentina1133 — APPROVED** [review]

File: tests/communities/test_communities_channels.py (hunk 2 above)
Comment (anastasiyaig) [reply tests/communities/test_communities_channels.py:115]: lol sorry

File: tests/communities/test_communities_channels.py (hunk 1 above)
Comment (anastasiyaig) [reply tests/communities/test_communities_channels.py:97]: stupid pycharm! :D

**Comment by anastasiyaig** [comment]
address that here https://github.com/status-im/desktop-qa-automation/pull/548

**Pull request closed without merging**